prow/config.yaml

```diff
@@ -219,7 +219,22 @@ tide:
   merge_label: tide/merge-method-merge
   context_options:
     # Treat unknown contexts as optional
```

Comment:
I'm not sure I understand what "contexts" are here.

Is it worth adding a link to documentation?

Do we know what this will end up blocking?

Reply:
This should not block anyone. I did ping all leads just in case, but this makes it so that all github actions are set to required pass rather than optional as they are now. Essentially instead of whitelisting which should be required this is making it so all are required to pass unless stated otherwise ie blacklist.